doc/psbt.md

```diff
@@ -0,0 +1,132 @@
+# PSBT Howto for Bitcoin Core
+
+Since Bitcoin Core 0.17, an RPC interface exists for Partially Signed Bitcoin
```

Comment:
13941: dashify

Reply:
Should we? or should we just leave this as is for now? How to dashify this is confusing to me

Reply:
Hmmm... why don't we just skip 13941 then?

Reply:
No... I don't think that's needed, because we are supporting psbt at this point..